### DISCUSSION TITLE
flatpak permission issues

### DISCUSSION THREAD

**Discussion by cjns1989** [johnfactotum/foliate#643] [Q&A]
I  reinstalled flatpak and I still cannot launch Foliate. 

```
% flatpak run com.github.johnfactotum.Foliate
bwrap: Can't mkdir parents for /run/host/usr: Permission denied
```

I have another flatpak'd application called pulseeffects and naturally I had to reinstall it since I had removed everything flatpak from the system… and after reinstalling this application still runs OK… Go figure.

Anyway… I found a *.deb of an old release of Foliate — 2.1.1… I believe… and this installed without a glitch and runs likewise. And now for all I know I have a keylogger, trojan, and what not running on my system and probably making calls to "servers in China"… Oh dear… oh dear…

Hopefully I'll be able to get rid of that flatpak junk for good when the next debian upgrade comes along. 

Oh… while mucking around with this I found an old epub I'd done a few years back… a song book with the score of the songs on one page and the lyrics on a page facing the sheet music. I used lilypond to do the scores and since I don't think epub supports that sort of thing I used png's. Of course these images must  be rendered full-screen because (1) that's the way song books are designed to be… and (2) reduced to something smaller especially on smaller tablets you'd need a maginifying glass to take a peek at the scores. Just another example (among many) where it would be nice if full-screen images scaled to the maximum WxH of the display. Turns the KOReader android app does this correctly… I thought I'd let you know.

Thanks, 

CJ

**Comment by johnfactotum** [comment]
I don't know what causes the problem, but a quick search leads to this issue: https://github.com/flatpak/flatpak/issues/1357
